addons/web/static/tests/widgets/company_switcher_tests.js

```diff
@@ -39,7 +40,14 @@ async function testSwitchCompany(assert, params) {
 async function testToggleCompany(assert, params) {
     assert.expect(2);
     var menu = await initMockCompanyMenu(assert, params);
-    await testUtils.dom.click(menu.$(`div[data-company-id=${params.company}] div.toggle_company`));
+    if (params.companies) { // if multiple companies are enabled/disabled
+        params.companies.forEach(async (company) => {
+            await testUtils.dom.click(menu.$(`div[data-company-id=${company}] div.toggle_company`));
+        });
+    } else {
+        await testUtils.dom.click(menu.$(`div[data-company-id=${params.company}] div.toggle_company`));
+    }
+    await concurrency.delay(1000);
```

Comment:
suggestion:
 - keep the helper as before
 - mock the delay and set it to 0 by default
 - in your new test, keep the real delay, and call the method twice to simulate quick consecutive clicks